README.md

```diff
@@ -20,7 +20,7 @@ You can still raise issues or pull requests if you want to. As soon as we’re a
 
 - **Accessibility**: Following WAI-ARIA best practices and testing with assistive technologies.
 - **User experience**: Supporting a wide variety of user needs.
-- **Compatibility**: Working with as many browsers, devices, and assistive technologies as possible.
+- **Compatibility**: Working with as many browsers, devices, and assistive technologies as possible. However, we're unable to support versions of Internet Explorer (IE) before IE11.
```

Comment:
Would this wording be less negative (because it says what we're able to do, instead of what we're unable to do)?

```suggestion
- **Compatibility**: Working with as many browsers, devices, and assistive technologies as possible. However, we only support versions of Internet Explorer (IE) from IE11 on.
```